### PR TITLE
modified login

### PR DESCRIPTION
**Pull request by KolluruBalaji123**
plesae review login component

### CODE REVIEW

File: src/Components/Login.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 function Login() {
   return (
     <div>
-      <h2>I am belong to Login page</h2>
+      <h2>I am belong to  to my page</h2>
```
Comment (naveen5005) [inline src/Components/Login.jsx]: to added two times, please remove it

Comment (KolluruBalaji123) [reply src/Components/Login.jsx]: Review Comments addrress

**Review by naveen5005 — APPROVED** [review]
looks good, approved.

**Comment by naveen5005** [comment]
merging navbar changes

**Pull request merged**